src/libpmemobj/pmalloc.c

```diff
@@ -551,6 +566,75 @@ CTL_READ_HANDLER(arena_id)(void *ctx,
 	return 0;
 }
 
+/*
+ * CTL_WRITE_HANDLER(arena_id) -- assigne the arena to the calling thread
+ */
+static int
+CTL_WRITE_HANDLER(arena_id)(void *ctx,
+	enum ctl_query_source source, void *arg, struct ctl_indexes *indexes)
+{
+	PMEMobjpool *pop = ctx;
+	unsigned arena_id = *(unsigned *)arg;
+
+	int ret = heap_set_arena_thread(&pop->heap, arena_id);
+	if (ret) {
+		ERR("arena with id: %u, does not exist", arena_id);
```

Comment:
LOG - ERR should be used only with external functions (syscalls, library calls);
comma after %u looks weird